internal/jobservice/eventstojobs/eventstojobs.go

```diff
@@ -82,6 +82,17 @@ func (eventToJobService *EventsToJobService) streamCommon(ctx context.Context, t
 			}
 		}()
 
+		log.Infof("GetJobEventMessage for %s/%s with id %s", eventToJobService.queue, eventToJobService.jobSetId, fromMessageId)
```

Comment:
Are Daniele and/or others comfortable with info-level logging on this, or would they prefer it to be debug-level?

Reply:
I think it would be fine as this should only be once every subscribe.  But we can always mess around with this.